zulip_botserver/tests/test_server.py

```diff
@@ -25,6 +25,13 @@ class MockLibModule:
         def handler_class(self) -> Any:
             return BotServerTests.MockMessageHandler()
 
+    def setUp(self) -> None:
+        # Since initializing Client invokes `get_server_settings` that fails in the test
+        # environment, we need to mock it to pretend that there exists a backend.
+        super().setUp()
+        self.patch = mock.patch("zulip.Client.get_server_settings", return_value=mock.Mock())
+        self.patch.start()
```

Comment:
It would likely be better for this to return fixture content that we fetch once from a real Zulip server and store in a file in the codebase.